app/controllers/concerns/secured.rb

```diff
@@ -51,15 +51,15 @@ def new_user?
   end
 
   def admin?
-    @current_user[:extra][:raw_info]['https://cloudnativedays.jp/roles'].include?("#{conference.abbr.upcase}-Admin")
+    conference && current_user[:extra][:raw_info]['https://cloudnativedays.jp/roles'].include?("#{conference.abbr.upcase}-Admin")
```

Comment:
This change is informed by the following tests:

* https://github.com/cloudnativedaysjp/dreamkast/blob/3c1a807a103bb5b8d4d5ac29fac5e56f23170b09/spec/requests/timetable_spec.rb#L78
* https://github.com/cloudnativedaysjp/dreamkast/blob/3c1a807a103bb5b8d4d5ac29fac5e56f23170b09/spec/requests/tracks_spec.rb#L91

In these tests, the target Conference does not exist, so render 404 Page.
During this rendering, the code attempts to display the login button in the header as shown here:

https://github.com/cloudnativedaysjp/dreamkast/blob/3c1a807a103bb5b8d4d5ac29fac5e56f23170b09/app/views/layouts/partial/_login_button.erb#L36-L40

However, since `current_user` is already logged in, `admin?` is called and raise an error.

> undefined method `abbr' for nil

Before this PR, `@current_user` was not a method call, so this `<% if @current_user %>` block would not be rendered.